### ISSUE TITLE
Surgery has terrible user feedback

### ISSUE THREAD

**Issue by WJohn** [tgstation/tgstation#19255]
Surgery, without a surgical computer, is absolutely awful because it gives you almost no feedback whatsoever.

You have no way to know what surgery is currently being performed. It does not show up with examining, checking yourself, sprites, or anything other than the surgical computer.

You have no way of knowing what the next surgical step is, and given that we both have a shitload of different kinds of surgery, each with different steps and amount of steps, it's very easy to get lost and difficult to pick up because of the above problem.

Knowing what works as a ghetto tool is impossible outside of either looking at the extremely outdated and sometimes downright wrong wiki, looking at the code, or just blindly testing for items that sound like they might work (but you usually end up chopping somebody up instead). There really should be a little tidbit in the descriptions of these items that state "Performs well as a ghetto bone saw for surgery" or "Performs poorly as a ghetto scalpel for surgery".

**Comment by Dorsisdwarf** [comment]
Feature Request

**Comment by InsaneHyena** [comment]
> You have no way of knowing what the next surgical step is
But this is wrong. Both robotics and surgery have a surgical computer, hell, even maint surgery on box has an (unfinished, but still) surgical computer that tells you what the next step is and what the surgery is. The only times when you don't have such help are when you're performing a ghetto surgery in a place not intended for it.

**Comment by WJohn** [comment]
I pointed this out several, SEVERAL times that *WITHOUT* a surgical computer, you have no way of knowing what the next step is. It doesn't tell you in the text after completing a step or anything.

**Comment by TechnoAlchemisto** [comment]
That's because you have to know what you're doing if you're going to perform a surgery without a computer

**Comment by ChangelingRain** [comment]
there's a lot of weird shit like the implant removal step, which requires either a retractor or a *crowbar* and in general it's very non-obvious what'll work for any given step

**Comment by dhostak** [comment]
But people aren't surgeons, it makes sense to not be super family fun time easy to do surgery

**Comment by Niknakflak** [comment]
The wiki is pretty out of date on surgery as well.

**Comment by jughu** [comment]
Use common sense wait.. that isnt common

**Comment by WJohn** [comment]
Common sense isn't doesn't apply, the wiki isn't reliable, winging it can kill your patient, and just remembering it from performing all the operations numerous times is pretty shitty to force the players to do.

**Comment by Armhulen** [comment]
whoa ghetto surgery is dangerous????
i kinda like that, you shouldn't be able to do surgery anywhere without risks

**Comment by Jalleo** [comment]
Really we should make surgery use some sort of table crafting system.

**Comment by WJohn** [comment]
Ghetto surgery already carries RNG fail chances where you instead cleave your patient. We don't need to make it awkward as shit and nearly impossible to figure it out on top of that.

**Comment by xxalpha** [comment]
I am pretty sure the computer does all of those things.

**Comment by WJohn** [comment]
how can you miss that i said *without* a computer like 6 times

*without*

**Comment by xxalpha** [comment]
You learn how to do it without a computer then. This is a game about learning things, too.

**Comment by jughu** [comment]
Yeah exactly i mean like your not a veteran pkayer when you play it the first time  and finding ghettto tools is just thinking smart

**Comment by WJohn** [comment]
That's not an excuse to be lacking feedback for basic things like this. This is QoL thing that should not be brushed away by going "veterans can handle it anyway".

**Comment by PKPenguin321** [comment]
To be fair newbies wont be doing ghetto surgery anyways most of the time

**Comment by Archie700** [comment]
If you don't know what are you going to do for the next step without a computer then you probably shouldn't be doing surgery without a computer in the first place.

**Comment by Manybones** [comment]
Thats one of the risks involved in ghetto surgery. Surgery shouldn't be easy to perform out of a prepared room.

**Comment by nicbn** [comment]
Port bay surgery when?

**Comment by RemieRichards** [comment]
Bay surgery will never be ported, its code is atrocious.

**Comment by Repukan** [comment]
@Cyberboss not a bug nor issue, feature request.

**Comment by nicbn** [comment]
No, this is an issue.

**Comment by Repukan** [comment]
It's a feature request

**Comment by Cyberboss** [comment]
It's an issue

**Comment by Repukan** [comment]
¯\_(ツ)_/¯

**Comment by PKPenguin321** [comment]
Would it be a cool fix for medical doctors to have explicit messages saying what the next step might be, while everybody else gets a more vague one?

**Comment by nicbn** [comment]
The surgery computer already does it @PKPenguin321

**Comment by PKPenguin321** [comment]
Yeah but this issue is still up so clearly something needs to be added

**Comment by Kaptinalexander** [comment]
I would like to note that the organ manipulation for augs has a surgery step that requires you to hold the tool in your other hand. And the step telling you to prepare electronics is just vague as hell.
